### PR TITLE
Task-1 Demonstrate scripting of insert and remove

### PR DESCRIPTION
**Pull request by erikbuck**

### CODE REVIEW

File: test/CommandControllerTest.cpp [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+TEST(CommandController, runScript0) {
+    WSU::Model::CommandController commandController;
+    commandController.runScript("{\"character\": \"a\", \"command\": \"insertCharacterAtIndex\", \"index\": 0}");
+    EXPECT_EQ("a", commandController.getCurrentStoredStringValue());
+}
+
+TEST(CommandController, runScript1) {
+    WSU::Model::CommandController commandController;
+    commandController.runScript("{\"character\": \"a\", \"command\": \"insertCharacterAtIndex\", \"index\": 0}{\"character\": \"-\", \"command\": \"insertCharacterAtIndex\", \"index\": 0}");
```
Comment (jcgoble3) [inline test/CommandControllerTest.cpp]: This script would be easier to read if each command was on a separate line, taking advantage of the fact that adjacent string literals are concatenated by the compiler in C++.

File: test/CommandControllerTest.cpp [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    EXPECT_EQ("-a", commandController.getCurrentStoredStringValue());
+}
+
+TEST(CommandController, runScript2) {
+    WSU::Model::CommandController commandController;
+    commandController.runScript("{\"character\": \"a\", \"command\": \"insertCharacterAtIndex\", \"index\": 0}{\"character\": \"-\", \"command\": \"insertCharacterAtIndex\", \"index\": 0}{\"character\": \"a\", \"command\": \"insertCharacterAtIndex\", \"index\": 0}{\"character\": \"!\", \"command\": \"insertCharacterAtIndex\", \"index\": 1}");
```
Comment (jcgoble3) [inline test/CommandControllerTest.cpp]: Same as above; even harder to read. Separate lines would be better for readability.

File: test/CommandControllerTest.cpp [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    EXPECT_EQ("a!-a", commandController.getCurrentStoredStringValue());
+}
+
+TEST(CommandController, runScript3) {
+    WSU::Model::CommandController commandController;
+    commandController.runScript("{\"character\": \"a\", \"command\": \"insertCharacterAtIndex\", \"index\": 0}{\"character\": \"-\", \"command\": \"insertCharacterAtIndex\", \"index\": 0}{\"character\": \"a\", \"command\": \"insertCharacterAtIndex\", \"index\": 0}{\"command\": \"removeCharacterAtIndex\", \"index\": 1}");
```
Comment (jcgoble3) [inline test/CommandControllerTest.cpp]: Ditto above.

**Review by jcgoble3 — CHANGES_REQUESTED** [review]
Implementation looks good, though the tests have some readability issues (see the comments there).

File: test/CommandControllerTest.cpp (hunk 1 above)
Comment (erikbuck) [reply test/CommandControllerTest.cpp]: Done

File: test/CommandControllerTest.cpp (hunk 3 above)
Comment (erikbuck) [reply test/CommandControllerTest.cpp]: Done

File: test/CommandControllerTest.cpp (hunk 2 above)
Comment (erikbuck) [reply test/CommandControllerTest.cpp]: Done

**Review by jcgoble3 — APPROVED** [review]
All looks good now.

**Pull request merged**